### DISCUSSION TITLE
QT5 vs QT6

### DISCUSSION THREAD

**Discussion by Keatah** [Stellarium/stellarium#3141] [Q&A] [answered]
Can someone explain the major differences (and which should be used) between QT5 & QT6?

I'm basically using Stellarium (at the present time) as a what's up tool, to see what the night sky looks like on any given day. To observe solar system bodies and their positions throughout the years. Currently not hooked to a telescope, but that will change sometime soon I'm sure. So what's the advantages/disadvantages for QT5 vs QT6?

**Comment by Keatah** [comment]
Never mind I think I found the important differences listed in the pdf manual.

**Reply by gzotti** [reply, marked as answer]
For telescope use, note esp. https://github.com/Stellarium/stellarium/wiki/Common-Problems-for-the-current-version#problems-with-telescopecontrol-plugin. The Qt6 version seems to have some problem, solvable by somebody with a telescope that should be supported and willingness to debug/fix on a bad weekend.
